swan-cern/values.yaml

```diff
@@ -58,10 +58,40 @@ swan:
       image:
         name: gitlab-registry.cern.ch/swan/docker-images/jupyter/swan-cern
         tag: v0.0.27
+      networkPolicy:
+        enabled: true
+        ingress:
+          # Allow user pods to receive incoming connections on the NodePort service port range.
+          # This supports use cases where servers run inside the user pod and need to receive
+          # connections (e.g. Spark, HTCondor via Dask).
+          - from:
+            - ipBlock:
+                cidr: 0.0.0.0/0
+                # Exclude connections from other pods in our cluster's pod network.
+                except:
+                  - 10.0.0.0/8
+                  - 172.16.0.0/12
+                  - 192.168.0.0/16
+            ports:
+            - protocol: TCP
+              port: 30000
+              endPort: 32767
+        egress:
+          # Allow connection from the user pod to the hadoop token generator to get Hadoop credentials.
+          - to:
+              - podSelector:
+                  matchLabels:
+                    app: hadoop-token-generator
+            ports:
+              - port: 80
     scheduling:
       userPods:
         nodeAffinity:
           matchNodePurpose: ignore
+    proxy:
+      chp:
```

Comment:
chp?

Reply:
Also, why do we enable the proxy netpols in swan-cern and not in swan?

Reply:
I would enable it for all components in the swan chart, and then in swan-cern I would add the part that is swan cern specific (Spark, HTCondor, hadoop token generator).

Reply:
Ah I see, I changed it now so CERN specific restrictions are in swan-cern and the general policies are in swan